lucille-core/src/test/java/com/kmwllc/lucille/stage/AddRandomStringTest.java

```diff
@@ -208,7 +208,38 @@ public void testNestedWithFile() throws StageException {
   }
 
   @Test
-  public void testInvalidConfig() {
+  public void testDefaultNumOfTerms() throws Exception {
+    Stage stage = factory.get("AddRandomStringTest/noMinOrMaxNum.conf");
+
+    // defaults to min/max of 1
+    Document doc = Document.create("doc");
+
+    stage.processDocument(doc);
+
+    assertTrue(doc.has("data"));
+    int data = doc.getInt("data");
+    // lower is inclusive, upper is exclusive
+    assertTrue(data >= 0 && data < 100);
```

Comment:
the name of this test method suggests we're testing how many terms are generated -- should be one -- but where are we asserting that? I suppose doc.getInt would fail on a string that doesn't represent a single int... but the test would be clearer if we did doc.getString() and looked at the string as such

Reply:
after we do doc.getString() can we do something like a split the string on " " and assert there's only 1 element found... to be explicit about what we're looking for in this test?